### PR TITLE
Install node/npm same as environment

### PR DESCRIPTION
**Pull request by cartermp**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Install and use custom node version
+        run: npm i -g npm@7
+      - name: Package restore
+        run: npm install
+      - name: Build docs
+        run: npm run build-docs
```
Comment (MangelMaxime) [inline .github/workflows/publish.yml:19]: `npm run deploy` is already build the documentation 

https://github.com/fable-compiler/fable-compiler.github.io/blob/31a4722da7d623bcd04d8428b808842514873f59/package.json#L18

What `npm run deploy` does his:

1. Call `npm run build`, which does the following:
    - `npx rimraf deploy` clean the deploy directory from any artifacts
    - `concurrently 'npm:build-sass' 'npm:build-docs' 'npm:build-fable'` execute all these tasks in parallel
1. Deploy on github pages `gh-pages -d deploy -b master`

In theory, we should remove `npm run build-docs` as it is already done in `npm run deploy` which clean the artifacts before building everything.

**Review by MangelMaxime — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (cartermp) [reply .github/workflows/publish.yml:19]: Should `npm start` also run `npm build-docs`? Then it wouldn't be needed in the gitpod definition, allowing people to easily see docs changes without setting it up themselves

Comment (cartermp) [reply .github/workflows/publish.yml:19]: Actually scratch that, they may still need to do that

**Comment by cartermp** [comment]
Should be good to go!

**Comment by alfonsogarciacaro** [comment]
Thanks!

**Comment by cartermp** [comment]
Looks like there's a little more work to do, it's likely some permissions stuff though.

**Comment by MangelMaxime** [comment]
Thank you for the help @cartermp

**Pull request merged**